src/BIAnalyticsService/CreateParticipantScreeningEpisode/CreateParticipantScreeningEpisode.cs

```diff
@@ -0,0 +1,108 @@
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using NHS.ServiceInsights.Common;
+using NHS.ServiceInsights.Model;
+
+namespace NHS.ServiceInsights.BIAnalyticsService;
+
+public class CreateParticipantScreeningEpisode
+{
+    private readonly ILogger<CreateParticipantScreeningEpisode> _logger;
+    private readonly IHttpRequestService _httpRequestService;
+    public CreateParticipantScreeningEpisode(ILogger<CreateParticipantScreeningEpisode> logger, IHttpRequestService httpRequestService)
+    {
+        _logger = logger;
+        _httpRequestService = httpRequestService;
+    }
+
+    [Function("CreateParticipantScreeningEpisode")]
+    public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequestData req)
```

Comment:
This function is dealing with creating a new participant screening episode so I'd suggest "post" for the HTTP method